README.md

```diff
@@ -50,6 +50,7 @@ Components in beta stage may include breaking changes, new features, and bug fix
 | [terra-application-menu-layout](https://github.com/cerner/terra-framework/tree/master/packages/terra-application-menu-layout) | [![NPM version](https://img.shields.io/npm/v/terra-application-menu-layout.svg)](https://www.npmjs.org/package/terra-application-menu-layout) | ![Stable](https://img.shields.io/badge/status-Stable-brightgreen.svg) |
 | [terra-application-name](https://github.com/cerner/terra-framework/tree/master/packages/terra-application-name) | [![NPM version](https://img.shields.io/npm/v/terra-application-name.svg)](https://www.npmjs.org/package/terra-application-name) | ![Stable](https://img.shields.io/badge/status-Stable-brightgreen.svg) |
 | [terra-application-utility](https://github.com/cerner/terra-framework/tree/master/packages/terra-application-utility) | [![NPM version](https://img.shields.io/npm/v/terra-application-utility.svg)](https://www.npmjs.org/package/terra-application-utility) | ![Stable](https://img.shields.io/badge/status-Stable-brightgreen.svg) |
+| [terra-dialog-modal](https://github.com/cerner/terra-framework/tree/master/packages/terra-dialog-modal) | [![NPM version](https://img.shields.io/npm/v/terra-dialog-modal.svg)](https://www.npmjs.org/package/terra-dialog-modal) | ![Stable](https://img.shields.io/badge/status-Stable-brightgreen.svg)|
```

Comment:
Nice! I think we usually forget to update this part 😄

Reply:
:D Looks like I forgot this in the notification-dialog PR. I'll update there